haskell/repl.bzl

```diff
@@ -203,7 +204,7 @@ def _create_HaskellReplCollectInfo(target, ctx):
         load_infos[target.label] = HaskellReplLoadInfo(
             source_files = hs_info.source_files,
             boot_files = hs_info.boot_files,
-            import_dirs = set.to_depset(hs_info.import_dirs),
+            import_dirs = depset(sets.to_list(hs_info.import_dirs)),
```

Comment:
Could this perhaps be another target for upstreaming?

Reply:
Good point. I've made a note of it and will add it to my Skylib PR.

Reply:
If we're planning on upstreaming that, I think the `to_depset` function should then still remain in our internal sets library, so that it's more conveniently replaceable everywhere it's used in the future.

Reply:
Done.